modules/developers-guide/pages/install/install_dse_68.adoc

```diff
@@ -13,6 +13,10 @@ is a simple method.
 xref:vm_dse_68[virtual machines or bare metal], use the method that downloads a zip file of
 Stargate jar files and install.
 
+CAUTION: At this time Stargate does not support advanced workloads such as Graph and Search. If you are 
```

Comment:
@polandll let me know if this would better fit somewhere else and if the `CAUTION` is appropriate.